File: codes/exp5.java
import java.io.*;

class time {
    int hour, min, sec;

    public time(int h, int m, int s) {
        hour = h;
        min = m;
        sec = s;
    }

    public time() {
        hour = 0;
        min = 0;
        sec = 0;
    }

    public time sum(time t2) {
        time t3 = new time();
        t3.sec = sec + t2.sec;
        t3.min = min + t2.min;
        t3.hour = hour + t2.hour;
        if (t3.sec >= 60) {
            t3.min = t3.sec / 60;
            t3.sec = t3.sec % 60;
        }
        if (t3.min >= 60) {
            t3.hour = t3.min / 60;
            t3.min = t3.min % 60;
        }
        return (t3);
    }

    public void display() {
        System.out.println(hour + ":" + min + ":" + sec);
    }
}

public class exp5 {
    public static void main(String args[]) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        System.out.print("Syeda Reeha Quasar 14114802719 \n\n");
        time t1 = new time(1, 56, 55);
        time t2 = new time(2, 00, 10);
        time t3 = new time();
        t3 = t1.sum(t2);
        System.out.print("Time 1:- ");
        t1.display();
        System.out.print("Time 2:- ");
        t2.display();
        System.out.print("Resultant time after addition is:- ");
        t3.display();
    }
}

File: codes/exp5.cpp
#include <iostream>
using namespace std;

class Time
{
    int hh, mm, ss;

public:
    Time() {}
    Time(int hh, int mm, int ss)
    {
        this->hh = hh;
        this->mm = mm;
        this->ss = ss;
    }
    void disp()
    {
        cout << hh << ":" << mm << ":" << ss;
    }
    void sum(Time t1, Time t2)
    {
        ss = t1.ss + t2.ss;
        mm = ss / 60;
        ss = ss % 60;
        mm = mm + t1.mm + t2.mm;
        hh = mm / 60;
        mm = mm % 60;
        hh = hh + t1.hh + t2.hh;
    }
};

int main()
{
    printf("Syeda Reeha Quasar 14114802719 \n");

    Time t1(2, 22, 34);
    cout << "The Time T1 Is: ";
    t1.disp();
    Time t2(4, 33, 50);
    cout << "\n\nThe Time T2 Is: ";
    t2.disp();
    Time t3;
    t3.sum(t1, t2);
    cout << "\n\nThe Resultant Time Is: ";
    t3.disp();
}